src/index/precision_locking_index/index.hpp

```diff
@@ -57,6 +67,18 @@ class HashTableWithPrecisionLockingIndex {
     range_index_.ForceInsert(key);
   }
 
+  bool Insert(const std::string_view key) {
+    bool inserted_range = range_index_.Insert(key);
+    if (!inserted_range) return false;
```

Comment:
Is it okay to insert into range_index first?
For semantics of PLI, I'd prefer data to be inserted into point_index first.

1. point_index_.Put(key)
2. range_index_.Insert(key)
  3. (if the insertion failed) delete record from point_index